proxy/src/com/trimlighthacking/BuiltInPatternSetRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.trimlighthacking;
```

Comment:
Consider com.knottysoftware.trimlight.proxy?

You should generally own the domain you use for your package name.